src/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -41,7 +41,7 @@ final class SearchFilter extends AbstractFilter implements SearchFilterInterface
 
     public const DOCTRINE_INTEGER_TYPE = Types::INTEGER;
 
-    public function __construct(ManagerRegistry $managerRegistry, IriConverterInterface $iriConverter, PropertyAccessorInterface $propertyAccessor = null, LoggerInterface $logger = null, array $properties = null, IdentifiersExtractorInterface $identifiersExtractor = null, NameConverterInterface $nameConverter = null)
+    public function __construct(ManagerRegistry $managerRegistry, IriConverterInterface $iriConverter, PropertyAccessorInterface $propertyAccessor = null, LoggerInterface $logger = null, array $properties = null, IdentifiersExtractorInterface $identifiersExtractor, NameConverterInterface $nameConverter = null)
```

Comment:
please revert this needs to be optional

Reply:
If i leave it optional, calling the identifiersExtractor in the SearchFilterTrait doesn't work - or at least it didn't while running my tests.